samples/core-projects/libraries/new-library/test/LibraryTests/project.json

```diff
@@ -1,14 +1,22 @@
 {
-	"commands": {
-		"test":"xunit.runner.dnx"
+	"version": "1.0.0",
+	"buildOptions": {
+		"debugType": "portable"
 	},
+	"testRunner": "xunit",
 	"frameworks": {
-		"dnxcore50":{
+		"netcoreapp1.0":{
 			"dependencies": {
-				"Library":"",
-				"System.Runtime":"4.0.0-rc1-*",
-				"xunit":"2.1.0",
-				"xunit.runner.dnx": "2.1.0-rc1-*"
+				"Library":{
+					"version": "1.0.0",
```

Comment:
Here's what happens when I omit the version:

<img width="1077" alt="screen shot 2016-07-07 at 10 58 01 am" src="https://cloud.githubusercontent.com/assets/6309070/16663653/c4b076f2-4431-11e6-88cb-aa7301836b04.png">